tests/integration/components/async-await-test.js

```diff
@@ -448,6 +448,48 @@ module('Integration | Component | async-await', function(hooks) {
 
         assert.dom().hasText('resolved value');
       });
+
+      module('with no Ember.onerror', function(hooks) {
+        hooks.beforeEach(function(assert) {
+          // NOTE: this gets reset in the outer module
+          Ember.onerror = undefined;
+
+          expectRejection = async (reason, callback) => {
+            let called = 0;
+            let _consoleAssert = console.assert; // eslint-disable-line no-console
+
+            try {
+              Ember.onerror = error => {
```

Comment:
hm, doesn't this defeat L455?

Reply:
whoops. Must've `ctrl+z`ed too far. This could say `console.assert = error...`

Reply:
I want to talk to you about how to reuse the code where we mock `Ember.onerror` since this is nearly identical.